app/controllers/orders_controller.rb

```diff
@@ -30,6 +29,17 @@ def show
 
   private
 
+  def set_doomsday_flash_message
+    if current_user && current_user.total_purchased >= 50000
+      flash_link = "#{view_context.link_to 'Doomsday', doomsday_path}"
+      flash[:alert] = {color: 'red', message: "Doomsday Hath Commeth: #{flash_link} "}
```

Comment:
Line is too long. [86/80]<br>Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.